tests/registry_interface.rs

```diff
@@ -474,6 +520,15 @@ mod interface_tests {
         tl4.insert("tag".to_string());
         tl4.insert("three".to_string());
         println!("Running check_tag_list_n_last 4");
-        check_tag_list_n_last(&client, 2, "latest", &tl4)
+        check_tag_list_n_last(&client, 2, "latest", &tl4);
+        
+        println!("Running get_readiness");
+        get_readiness(&client);
+
+        println!("Running get_health");
+        get_health(&client);
+
+        println!("Running get_metrics");
```

Comment:
can we add a test to make sure that the manifest and blob request numbers go up by the expected numbers? Just scrape the endpoint and save the numbers, then run again after getting a manifest .